app/presenters/reservation_user_action_presenter.rb

```diff
@@ -23,7 +23,7 @@ def user_actions
 
     if can_switch_instrument?
```

Comment:
Does this ever return true for the operator? Also, it's hidden here, but are we hiding the accessories link as well from the operator?